meinberlin/apps/embed/templates/meinberlin_embed/embed.html

```diff
@@ -6,7 +6,7 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title>Embedded meinBerlin</title>
-    <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
+    <link rel="shortcut icon" type="image/x-icon" href="{% static 'images/favicon.ico' %}?v=2" />
```

Comment:
@rmader This was part of the attempt to fix @phillimorland 's local email sending. Should it stay like that or go back to the old form?

Reply:
Uh that looks weird. Definitely should not stay like that :/

Reply:
Ups, sorry, I read it the wrong way around. It looks absolutely like it should be that way!